src/palace/manager/service/celery/celery.py

```diff
@@ -129,9 +129,9 @@ def beat_schedule() -> dict[str, Any]:
             "task": boundless.reap_all_collections.name,
             "schedule": crontab(
                 day_of_week="6",
-                minute="0",
+                minute="8",
                 hour="4",
-            ),  # Every Saturday at 4:00 AM
+            ),  # Every Saturday at 4:08 AM
```

Comment:
I offset this slightly, so the reaper and import don't run at the exact same time.